docs/references/badges.md

```
@@ -0,0 +1,145 @@
+---
+title: Badges & Statuses
+description: Label pages with badges like ALPHA or WIP — ad-hoc via `badge`, named via the `status` registry — rendered in the sidebar and breadcrumb.
+status: new
+---
+
+# Badges & Statuses
+
+Badges are small labels — `ALPHA`, `WIP`, `Deprecated` — attached to a page. They render in **both** the sidebar (next to the page's nav item) and the breadcrumb (right of the trail). Attach one two ways:
+
+- **`badge`** — an ad-hoc label defined inline on the page.
+- **`status`** — a reference to a named, documented preset in the [status registry](#statuses).
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)